config.py

```
@@ -0,0 +1,381 @@
+    }
+}
+
+# ============================================================================
+# UTILITY FUNCTIONS
+# ============================================================================
+
+def get_app_data_dir() -> Path:
+    """Gibt das App-Datenverzeichnis zurück."""
+    if os.name == 'nt':  # Windows
+        base_dir = os.environ.get('APPDATA', '')
+    elif sys.platform == 'darwin':  # macOS
```

Ruff (0.12.2)

Undefined name `sys`

(F821)